build_tools/benchmarks/mmperf/run_mmperf.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Runs SHARK tank using both SHARK-Runtime and IREE-Runtime, producing benchmark
+# numbers.
+#
+# Usage:
+#    ./run_shark.sh \
+#        <SHA of https://github.com/nod-ai/SHARK.git to pin to> \
+#        <pytest regex> e.g. "cpu", "cuda", "cuda and torch".
+#        <driver> e.g. "cpu", "cuda", "vulkan"
+#        <output directory>
+
+set -xeuo pipefail
+
+export REPORT_DIR=$1
+# Either `cpu` or `cuda`.
+export BACKEND=$2
+
+git clone https://github.com/mmperf/mmperf.git
+cd mmperf
+
+# Update IREE.
+cd external/iree
+git fetch --all
```

Comment:
Do you actually need `--all` here? You can usually just fetch the specific branches/tags/refs for a single remote

Reply:
Good point, removed the --all.